core/src/test/java/com/ericsson/bss/cassandra/ecchronos/core/repair/state/TestAlarmPostUpdateHook.java

```diff
@@ -0,0 +1,271 @@
+        myClock.setTime(start);
+
+        myPostUpdateHook.postUpdate(myRepairStateSnapshot);
+
+        // verify - repaired
+        verify(myFaultReporter).cease(eq(RepairFaultReporter.FaultCode.REPAIR_WARNING), eq(expectedData));
+    }
+
+    @Test
+    public void testThatErrorAlarmIsSentAndCeased()
+    {
+        // setup - not repaired
+        long daysSinceLastRepair = GC_GRACE_DAYS;
+        long start = System.currentTimeMillis();
+        long lastRepaired = start - TimeUnit.DAYS.toMillis(daysSinceLastRepair);
+
+        Map<String, Object> expectedData = new HashMap<>();
+        expectedData.put(RepairFaultReporter.FAULT_KEYSPACE, keyspaceName);
+        expectedData.put(RepairFaultReporter.FAULT_TABLE, tableName);
+
+        // mock - not repaired
+        doReturn(lastRepaired).when(myRepairStateSnapshot).lastRepairedAt();
+        myClock.setTime(start);
+
+        myPostUpdateHook.postUpdate(myRepairStateSnapshot);
+
+        // verify - not repaired
+        verify(myFaultReporter).raise(eq(RepairFaultReporter.FaultCode.REPAIR_ERROR), eq(expectedData));
+
+        // setup - repaired
+        lastRepaired = start;
+        start = System.currentTimeMillis();
+
+        // mock - repaired
+        doReturn(lastRepaired).when(myRepairStateSnapshot).lastRepairedAt();
+        myClock.setTime(start);
+
+        myPostUpdateHook.postUpdate(myRepairStateSnapshot);
+
+        // verify - repaired
+        verify(myFaultReporter).cease(eq(RepairFaultReporter.FaultCode.REPAIR_WARNING), eq(expectedData));
+    }
+
+    @Test
+    public void testThatErrorAlarmIsSentAndCeasedExternalRepair()
+    {
+        // setup - not repaired
+        long daysSinceLastRepair = GC_GRACE_DAYS;
+        long start = System.currentTimeMillis();
+        long lastRepaired = start - TimeUnit.DAYS.toMillis(daysSinceLastRepair);
+
+        Map<String, Object> expectedData = new HashMap<>();
+        expectedData.put(RepairFaultReporter.FAULT_KEYSPACE, keyspaceName);
+        expectedData.put(RepairFaultReporter.FAULT_TABLE, tableName);
+
+        // mock - not repaired
+        doReturn(lastRepaired).when(myRepairStateSnapshot).lastRepairedAt();
+        myClock.setTime(start);
+
+        myPostUpdateHook.postUpdate(myRepairStateSnapshot);
+
+        // verify - not repaired
+        verify(myFaultReporter).raise(eq(RepairFaultReporter.FaultCode.REPAIR_ERROR), eq(expectedData));
+
+        // setup - repaired
+        lastRepaired = start;
+        start = System.currentTimeMillis();
+
+        // mock - repaired
+        doReturn(lastRepaired).when(myRepairStateSnapshot).lastRepairedAt();
+        myClock.setTime(start);
+
+        myPostUpdateHook.postUpdate(myRepairStateSnapshot);
+
+        // verify - repaired
+        verify(myFaultReporter).cease(eq(RepairFaultReporter.FaultCode.REPAIR_WARNING), eq(expectedData));
+    }
+
+    @Test
+    public void testThatAlarmIsNotSentWhenGcGraceIsBelowRepairInterval()
```

Comment:
I think this test case could be removed. It used to test a legacy behavior which hasn't existed for a while.